models/etcd/index.go

```diff
@@ -152,8 +153,10 @@ func (mi *machinesIndex) query(q sabakan.Query) []string {
 
 	res := make(map[string]struct{})
 
-	for _, serial := range mi.Rack[q.Rack()] {
-		res[serial] = struct{}{}
+	for _, rack := range strings.Split(q.Rack(), ",") {
```

Comment:
Why can we only specify multiple racks?
We want to specify other options as well.

The original issue is just an example of a rack.